default/stringtables/en.txt

```diff
@@ -1515,6 +1515,9 @@ Limit of client message size in bytes.
 OPTIONS_DB_DROP_EMPIRE_READY
 Drop empire's readiness on joining to the playing game.
 
+OPTIONS_DB_TAKE_OVER_AI
+Allow human player to delegate any AI player.
```

Comment:
maybe clarify that this is just a server option...

Reply:
Fixed